x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/detection_alerts/assignments/assignments.cy.ts

```diff
@@ -54,6 +53,7 @@ describe.skip('Alert user assignment - ESS & Serverless', { tags: ['@ess', '@ser
     login(ROLES.soc_manager);
     login(ROLES.detections_admin);
     login(ROLES.platform_engineer);
+    login();
```

Comment:
Is this `login` method needed? Theoretically I believe it should not, in all the tests where the `login()` has been added to the before hook, we are calling on the beforeEach, `loadPageAs()`. As part of the implementation, that method, a login is performed so having that one sounds redundant. If it is needed, it would be nice to understand why to make sure we are not masking a possible issue :)

Reply:
Right, that's a good point! We do not really need `login()` here.